src/Fixer/ControlStructure/NoAlternativeSyntaxFixer.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ControlStructure;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Class NoAlternativeSyntaxFixer.
+ *
+ * @author Eddilbert Macharia <edd.cowan@gmail.com>
+ */
+final class NoAlternativeSyntaxFixer extends AbstractFixer
+{
+    const SINGLE_COLON = ':';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Replace control structure alternative syntax to use braces.',
+            [
+                new CodeSample(
+                    "<?php\nif(true):echo 't';else:echo 'f';endif;\n"
+                ),
+                new CodeSample(
+                    "<?php\nwhile(true):echo 'red';endwhile;\n"
+                ),
+                new CodeSample(
+                    "<?php\nforeach(array('a') as \$item):echo 'xc';endforeach;\n"
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound(
+            [
+                T_IF,
```

Comment:
```php
<?php if (1) { echo 2; }
```

is matching the condition, but fixer has nothing to do here.

check only that alternate syntax ending is present, thus `T_END*` tokens